src/main.rs

```diff
@@ -0,0 +1,23 @@
+extern crate afl;
```

Comment:
The tool that I used to generate these crashes is [afl](http://lcamtuf.coredump.cx/afl/). In particular, I used [afl.rs](https://github.com/frewsxcv/afl.rs). What you're seeing here is the runner for afl. It shouldn't be included in this PR.